server/SwipeDecision.py

```diff
@@ -0,0 +1,69 @@
+import mysql.connector
+from mysql.connector import errorcode
+from connect import connectToDB
+
+
+def getPotentialMatch(currentUserId):
+    try:
+        connection = connectToDB()
+        if(connection != False):
+            cursor = connection.cursor(buffered=True)
+            PotentialMatchQuery = "SELECT shownUser FROM PotentialMatch WHERE currentUser = %s and matchDecision IS NULL"
+            userID = (currentUserId,)      
```

Comment:
is that comma supposed to be there?

Reply:
ya supposed to be there for SQL to work.